vivisect/tests/testvivisect.py

```diff
@@ -549,7 +558,6 @@ def test_vaset_populate(self):
                                  (0x805a314, 0x805d87c, 'vdir', 'time_args'),
                                  (0x805a318, 0x805d880, 'vdir', 'time_args'),
                                  (0x805a31c, 0x805d886, 'vdir', 'time_args'),
-                                 (0x805a324, 0x0, 'vdir', 'format_types'),
```

Comment:
`format_types` is actually a structure (not NULL pointer):
![image](https://github.com/user-attachments/assets/bc72a9a9-315d-4e16-b56f-5e9c6abbcf93)
![image](https://github.com/user-attachments/assets/b632e888-6e1f-4666-a648-8dcbe1e2789b)

same with `indicator_style_types`, `time_style_types`, `default_tuning`, and `quote_style_vals`

dso_handle is a number, not a pointer: https://stackoverflow.com/a/39521737

and lots of zero-initialized numbers, not pointers:
![image](https://github.com/user-attachments/assets/1d08f790-adf9-4504-a155-677c40bf9a23)

`__FRAME_END__` is not a pointer, but just a location in the program.

Reply:
having written much of the Viv ELF parsing code, i would agree with you.

and as far as unit-tests are concerned, some of Vivisect's tests are real and trust-worthy, but arguably most of them are "shit changed and we didn't expect it" alarms.  something to give coverage and an indicator if we accidentally break something.  so if you're concerned about the unit-tests needing to be changed, all i need is a justification that the new answer is "more" correct or a desired side-effect of improving the Viv code.

Reply:
this is *especially* true for `redxor.stability.json` and other `stability` tests